lib/vcloud/launcher/cli.rb

```diff
@@ -1,113 +0,0 @@
-require 'optparse'
-
-module Vcloud
-  module Launcher
-    class Cli
-
-      def initialize(argv_array)
-        @config_file = nil
-        @usage_text = nil
-        @options = {
-          "dont-power-on"     => false,
-          "continue-on-error" => false,
-          "quiet"             => false,
-          "post-launch-cmd"   => false,
-          "verbose"           => false,
-        }
-
-        parse(argv_array)
-      end
-
-      def run
-        begin
-          launch = Vcloud::Launcher::Launch.new(@config_file, @options)
-          launch.run
-        rescue => error_msg
-          $stderr.puts(error_msg)
-          exit 1
-        end
```

Comment:
Discussed this with @bazbremner IRL; the change to use Trollop would currently output a backtrace rather than an exception message. He's pushing a commit to fix.